Sources/UnionNotifications/IOS18NotificationDialogContent.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  IOS18NotificationDialogContent.swift
+//  union-notifications
+//
+//  Created by RJ Kigner on 9/29/25.
+//
+
+import Foundation
+import SwiftUI
+
+struct  IOS18NotificationDialogContent: View {
+    let appName: String
+    let shouldEnableScroll: Bool
+    let maxContentHeight: CGFloat?
+    
+    private var localizedTitle: String {
+        String(format: String(localized: "\"%@\" Would Like to Send You Notifications", bundle: .module), appName) 
+    }
+    
+    var body: some View {
+        Group {
+            if shouldEnableScroll {
+                ScrollView {
+                    contentText
+                }
+                .frame(maxHeight: maxContentHeight)
+            } else {
+                contentText
+            }
+        }
+    }
+    
+    private var contentText: some View {
+        VStack(spacing: 4) {
+            Text(LocalizedStringKey(localizedTitle))
+                .font(.body.weight(.semibold))
+                .foregroundColor(.primary)
+                .multilineTextAlignment(.center)
+                .lineSpacing(2)
+                .fixedSize(horizontal: false, vertical: true)
+
+            Text("Notifications may include alerts, sounds, and icon badges. These can be configured in Settings.", bundle: .module)
```

Comment:
Set a maximum column length for your editor (something like 80 or 100 or 120) and make sure to adhere to it. You can look up multiline strings if needed.